### PR TITLE
The Role of Artificial Intelligence for Smarter Cybersecurity

### LINKED ISSUE #540: The Role of Artificial Intelligence for Smarter Cybersecurity
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.
Artificial intelligence (AI) and machine learning (ML) are playing an increasing role in cybersecurity. Security tools are analyzing data from millions of cyber incidents, and using it to identify potential threat. 
### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
1. How machine learning and artificial intelligence improve cybersecurity through:
 a)Network security
 b)Data centers
c) Threat hunting
d) Vulnerability management 
2. Negative effect of AI on cybersecurity 
3. The future of AI and Cybersecurity 
### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)

### PR DESCRIPTION
**Pull request by EKahuha**
This fixes #540 
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      

#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tool:** [Grammarly](https://grammarly.com)
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by EKahuha** [comment]
This fixes #540

File: articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+Organizations continue to embrace the Internet of Things (IoT), the cloud, and mobile technology. This has influenced considerable changes in the threat landscape and created more vulnerability points. Cybercriminals are leveraging these new vulnerability points to develop and launch sophisticated, high-volume, multi-dimensional attacks. Such attacks mean that data is at risk, and organizations must spend valuable time analyzing potentially malicious files.
+
+Using artificial intelligence software, organizations can process large volumes of threat data and adequately prevent and respond to breaches and hacks. This article will discuss how machine learning and artificial intelligence improve cybersecurity in four important areas: network security, data centers, threat hunting, and vulnerability management.
+
+### Network Security
+
+Network security mainly focuses on the understanding of network topography and creating a security policy of an organization. Securities policies can help to distinguish legitimate network connections from illegitimate ones. But drafting and maintaining policies for multiple networks may be challenging.
```
Comment (lalith1403) [inline articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md:7]: Securities policies can help to-> Security policies help distinguish

File: articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Often, some organizations may not have a precise naming convention for workloads and applications. This means that network security teams have to spend significant time attaching different workloads to given applications.
```
Comment (lalith1403) [inline articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md:9]: Can you clarify the situation in a bit more detail? I want to understand how did we land at precise naming conventions and how that affects security policies? 

Some information is needed for better flow between the previous and current paragraph.

File: articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+These activities are time-consuming, and the use of artificial intelligence technology can come in handy. AI can expedite these processes by learning and observing traffic patterns and suggesting security policies. This way, organizations save on time, resources, and effort. The organization can allocate these to other areas of technological advancement or development.
+
+### Data Centers
+
+Cyber attackers target [data centers](https://www.cisco.com/c/en/us/solutions/data-center-virtualization/what-is-a-data-center.html) in an attempt to obtain critical data. Cybersecurity has improved over the years with advancements in technology. Being aware of this, cybercriminals regularly develop more advanced malware strains to ensure successful cyber-attacks on organizations&#39; networks.
+
+Organizations can deploy artificial intelligence technology in their data centers to enhance data security. To ensure data security, AI essentially learns normal network behavior. It then detects cyber threats with regard to the variations from the regular network behavior.
```
Comment (lalith1403) [inline articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md:17]: I feel we are describing the role of AI very vaguely here. What are normal patterns of behavior? How are they defined? What type of AI solves this? As a reader, I want to know as much as possible from the article. 

Any person reading this knows about AI being used everywhere. What the reader might lack will be the field-knowledge. If you can talk about detecting shady behavior using an RNN or rule-based expert systems, it adds more value to the reader.

File: articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+The application of AI in a data center can help identify security loopholes and detect malware in data center systems. Most importantly, AI-based cybersecurity can analyze or screen outgoing and incoming data security threats.
+
+### Threat Hunting
+
+The asymmetric nature of cybersecurity is an inherent problem that seems to empower the acts of attackers. With this state, cyber-attackers need to only be successful once by exploiting just a single weakness while the defenders must be successful every time. Preventing threats requires more than just ensuring security data. It calls for combing threats on an entire [information technology stack](https://www.pcmag.com/encyclopedia/term/technology-stack).
+
+In this case, AI is essential to identify abnormalities, outliers, and patterns in all data without the need for applying fixed rules. Human analysts depend on this output for investigation and action. This is what is often referred to as threat hunting in security language. It involves narrowing down threats by collaborating security analytics and machine intelligence, and advanced human cognition.
```
Comment (lalith1403) [inline articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md:25]: Again saying AI is performing threat hunting and passing it down to the security team is on the surface-level, What we want to provide the reader with is value. How is AI performing threat hunting? Give an intuitive example, if possible. I am not asking you to describe the algorithm in detail. But what algorithms are we talking about and how do they fit in the overall problem of threat hunting.

File: articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+### Vulnerability Management
+
+Every year, there are many reported vulnerabilities to cybersecurity. It is extremely difficult to manage these unique vulnerabilities with traditional technology or human resources. Systems that use machine learning and artificial intelligence do not wait for online attackers to exploit existing vulnerabilities. Rather, these systems are proactive in finding potential vulnerabilities in information systems.
```
Comment (lalith1403) [inline articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md:29]: How is vulnerability different from threat hunting? I feel we are emphasizing the point that AI solves the problem. But, as a reader, I don't understand how is it working? Divide the explanation into the following steps:
1. Problem statement: What are we trying to solve?
2. How is AI solving it(mention a few algorithms, techniques, models, if any)?
3. Advantages of AI over other systems
4. How AI integrates with the problem and solves it.

File: articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+Besides, advanced hacking campaigns can use machine learning technology to manipulate routine tasks relying on artificial intelligence.
+
+AI can identify irregularities and abnormalities in a network, deduce patterns from data, and analyze user behaviors. With this information, identifying cyber vulnerabilities becomes easier. However, cyberattackers can provide malicious cyber programs that imitate legitimate AI-based algorithms, which means that the information might be susceptible to cybercrime.
+
+### The Future of AI and Cybersecurity
+
+The future of the application of artificial intelligence to improve cybersecurity looks promising. Security teams expect AI to enhance and respond better to cyber-attacks. Human analysts are designing better defense strategies, and AI will replicate the best methods to ensure cybersecurity. Response speed to threats will improve, and human experts will be freed up to handle more complicated investigations.
```
Comment (lalith1403) [inline articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md:45]: response speed -> response time.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
@EKahuha Good article. I feel the article is structured well. However, we need to provide more information to the user. I have mentioned a few places where additional information will help deliver tremendous value to the reader.

Excited to read the updated article.

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by EKahuha** [comment]
@lalith1403 I have updated this article.

**Review by lalith1403 — APPROVED** [review]

**Comment by hectorkambow** [comment]
@EKahuha 
Was wondering if you had any other sources to add to this article. I only see two ? 

Where was all this information sourced from.? 

We'd like to add all the sources under a Addtional resources section.

**Comment by EKahuha** [comment]
> @EKahuha
> Was wondering if you had any other sources to add to this article. I only see two ?
> 
> Where was all this information sourced from.?
> 
> We'd like to add all the sources under an Addtional resources section.

@hectorkambow 
Here are other resources: 
1. blog.eccouncil.org/the-role-of-ai-in-cybersecurity/#:~:text=Combining%20the%20strength%20of%20artificial,resulting%20in%20reduced%20response%20times.
2. https://www.balbix.com/insights/artificial-intelligence-in-cybersecurity/
3. https://itchronicles.com/security/how-ai-is-changing-the-cybersecurity-landscape/

**Comment by hectorkambow** [comment]
@EKahuha  
Thank you v much. 

Lets be sure to add all our sources in future articles (we prefer them as links within the article - like you have done). 

Just adds more credibility and if someone wants more info they can easily get to it.

**Comment by EKahuha** [comment]
> @EKahuha
> Thank you v much.
> 
> Lets be sure to add all our sources in future articles (we prefer them as links within the article - like you have done).
> 
> Just adds more credibility and if someone wants more info they can easily get to it.

Noted. Thank you @hectorkambow

**Comment by hectorkambow** [comment]
👋  Good evening/morning Jason, 

Wanted to ask if you could give this "Artificial Intelligence for Smarter Cybersecurity" article a review? 
It's been reviewed by one of our peer reviewers - but I saw the Security aspect - and thought it could use the Wise 👁️ of @jstangroome  regarding anything we may have missed before we publish. 

Thank you v much 🚀

**Review by jstangroome — APPROVED** [review]
Looks good to me

**Pull request merged**